src/EditorFeatures/CSharpTest/Formatting/FormattingEngineTests.cs

```diff
@@ -1970,6 +1970,41 @@ void MyMethod()
             await AssertFormatWithBaseIndentAsync(expected, code, baseIndentation: 4);
         }
 
+        [WpfFact]
+        [Trait(Traits.Feature, Traits.Features.Formatting)]
+        [WorkItem(27268, "https://github.com/dotnet/roslyn/issues/27268")]
+        public async Task PositionalPattern()
+        {
+            var code = @"[|
+class MyClass
+{
+    void MyMethod()
+    {
+        var point = new Point (3, 4);
```

Comment:
@gafter I have a couple questions:

1. Can the parentheses be empty?
2. Right now I account for the option to have a space between a method call and the opening parenthesis. Should it also account for options to place spaces inside method call parentheses?

Reply:
1. Yes, the parens can be empty, can contain just one pattern, or any number of comma-separated patterns.
2. I would expect the options that apply to the object creation expression to equally apply to the positional part of a pattern.  I think that is a yes.

---
In reply to: [246495924](https://github.com/dotnet/roslyn/pull/32286#discussion_r246495924) [](ancestors = 246495924)

Reply:
Thanks, I updated the formatter to treat the recursive pattern like a method call.